code-challenges.test.js

```diff
@@ -10,37 +10,161 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+
+
+
 // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
 // a) Create a test with an expect statement using the variable provided.
+describe("sentenceArr", () => {
+  it("takes in an array of objects and returns an array with a sentence about each person with their name capitalized", () => {
+    const people = [
+      { name: "ford prefect", occupation: "a hitchhiker" },
+      { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+      { name: "arthur dent", occupation: "a radio employee" }
+    ]
+    // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+  expect(sentenceArr(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+  })
+})
+
+// FAIL  ./code-challenges.test.js
+// sentenceArr
+//   ✕ takes in an array of objects and returns an array with a sentence about each person with their name capitalized (2 ms)
+
+// ● sentenceArr › takes in an array of objects and returns an array with a sentence about each person with their name capitalized
+
+//   ReferenceError: sentenceArr is not defined
 
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
 // b) Create the function that makes the test pass.
 
+/* PSEUDOCODE
+  - create a function named sentenceArr
+  - input: array of objects
+  - process:
+    - make the name value an array separating each word into its own index using split(" ")
```

Comment:
Be sure to include every step your code is taking! There is no mention of using a for loop to go through the new array you made from the value.name.split(" ") and capitalizing each first letter!